src/Microsoft.DocAsCode.Metadata.ManagedReference/ExtractMetadata/CompilationUtility.cs

```diff
@@ -17,6 +18,7 @@ namespace Microsoft.DocAsCode.Metadata.ManagedReference
     internal static class CompilationUtility
     {
         private static readonly Lazy<MetadataReference> MscorlibMetadataReference = new Lazy<MetadataReference>(() => MetadataReference.CreateFromFile(typeof(object).Assembly.Location));
+        private static readonly Lazy<MetadataReference> SystemMetadataReference = new Lazy<MetadataReference>(() => MetadataReference.CreateFromFile(typeof(EditorBrowsableAttribute).Assembly.Location));
 
```

Comment:
Is this actually referencing `System.Runtime.dll`? What about rename it to `SystemRuntimeReference`?

Reply:
It's referencing `System.dll`